packages/firestore/src/core/query.ts

```diff
@@ -568,7 +595,10 @@ export class RelationFilter extends Filter {
 
   isInequality(): boolean {
     return (
-      this.op !== RelationOp.EQUAL && this.op !== RelationOp.ARRAY_CONTAINS
+      this.op !== RelationOp.EQUAL &&
+      this.op !== RelationOp.ARRAY_CONTAINS &&
+      this.op !== RelationOp.IN &&
+      this.op !== RelationOp.ARRAY_CONTAINS_ANY
```

Comment:
When there was only one (or two) exceptions, it probably made sense to write it this way, but at this point we should probably flip the logic and write
```
return (this.op === RelationOp.LESS_THAN || this.op === RelationOp.LESS_THAN_OR_EQUAL || ...);
```
or perhaps:
```
return [RelationOp.LESS_THAN, RelationOp.LESS_THAN_OR_EQUAL, ...].indexOf(this.op) >= 0;
```